README.md

```diff
@@ -9,7 +9,12 @@ Meteor-electron strives to be the easiest way to create a desktop Meteor applica
 
 `meteor add quark:electron`
 
-After adding the package, an electron app wrapper will be while you develop.
+After adding the package, an electron app wrapper will be built and opened.
+
+This app, as well as the ready-to-distribute binaries (see [Deploy](#deploy)), is built within
+`YOUR_PROJECT_DIRECTORY/.meteor-electron`. This allows the apps to be easily located as well as
+(forthcoming) the apps to be cached for speedier startup. You should add this directory to your
+`.gitignore`.
```

Comment:
Not sure if it's better to do this in a tmp directory (that will occasionally be deleted and cause extra builds) or a new directory within the project that requires the user to update their `.gitignore`